instructure/canvas-android
# [MBL-16995][Student] Score to letter grade conversion

Test plan: 
- Verify that the scores are correctly converted to letter grade on all the affected screens.
- Verify that we never show quantitative data when it is restricted.
- Verify that we don't convert the scores when quantitative data is not restricted.
- Compared grades with the web for reference.

refs: MBL-16995
affects: Student
release note: Added letter grade conversion when quantitative data is restricted.

## Checklist
